### PR TITLE
Add more definitions for Linux/SystemZ.

### PR DESCRIPTION
**Pull request by redstar**
Adds ucontext, setjmp and inotify.

### CODE REVIEW

File: src/core/sys/posix/setjmp.d [hunk 1]
```diff
@@ -121,6 +121,15 @@ version( CRuntime_Glibc )
...
         }
     }
+    else version (SystemZ)
+    {
+        struct __s390_jmp_buf
+        {
+            c_long[10] __gregs;
+            c_long[8] __fpregs;
+        }
+        alias __s390_jmp_buf[1] __jmp_buf;
```
Comment (ibuclaw) [inline src/core/sys/posix/setjmp.d]: The preferred style is ``alias __jmp_buf = __s390_jmp_buf[1];``, no?

Comment (redstar) [inline src/core/sys/posix/setjmp.d]: I changed it.

**Comment by ibuclaw** [comment]
OK

**Comment by ibuclaw** [comment]
Auto-merge toggled on

**Pull request merged**